output-tests/README.md

```diff
@@ -0,0 +1,26 @@
+These tests are intended to validate that implementations are correctly generating output in accordance with the specification.
+
+Output was initially specified with draft 2019-09.  It remained unchanged for draft 2020-12 <!-- do we need explicit tests for 2002-12? -->, but will receive an update with the next release.
```

Comment:
On explicit tests for 2020-12, personally I think ideally yes but it's fine not to have them initially, otherwise someone with an implementation only supporting 2020-12 (which we even encourage these days?) has no way of running these.

But doesn't seem top priority relative to just getting some tests in.

Reply:
Okay. It's just going to be an exact copy of 2019.  Makes maintaining the tests more difficult.

Reply:
Could always just symlink the directory too.

Reply:
Oh, we can't though, since the `$schema`s will be different, so not precisely the same...

Reply:
Ugh.  Fair point.